### PR TITLE
Added powerstore docs

### PR DESCRIPTION
**Pull request by prajwalpatil25**

### CODE REVIEW

File: content/docs/storage/platforms/powerstore/Product_Guide.md [hunk 1]
```diff
@@ -0,0 +1,1593 @@
...
+            <br>
+        </td>
+        <td>
+            <br>
+        </td>
+    </tr>
+    <tr>
+        <td>capacity_unit</td>
+        <td>The Capacity Unit corresponding to the size.</td>
+        <td>Optional and computed</td>
```
Comment (AnikaAgiwal2711) [inline content/docs/storage/platforms/powerstore/Product_Guide.md]: computed , C should be in capital.

File: content/docs/storage/platforms/powerstore/Product_Guide.md [hunk 2]
```diff
@@ -0,0 +1,1593 @@
...
+    *   [Synopsis](#synopsis)
+    *   [List Of Parameters](#list-of-parameters)
+    *   [Examples](#examples)
+    *   [Notes](#notes)
+*   [SnapshotRule Resource](#snapshotrule-resource)
+    *   [Synopsis](#synopsis-1)
+    *   [List Of Parameters](#list-of-parameters-1)
+    *   [Examples](#examples-1)
+    *   [Notes](#notes-1)
+*   [ProtectionPolicy Resource](#protectionpolicy-resource)
```
Comment (AnikaAgiwal2711) [inline content/docs/storage/platforms/powerstore/Product_Guide.md:22]: I Think it should be Protection Policy and not ProtectionPolicy

File: content/docs/storage/platforms/powerstore/Product_Guide.md [hunk 3]
```diff
@@ -0,0 +1,1593 @@
...
+    *   [Synopsis](#synopsis-2)
+    *   [List Of Parameters](#list-of-parameters-2)
+    *   [Examples](#examples-2)
+    *   [Notes](#notes-2)
+*   [StorageContainer Resource](#storagecontainer-resource)
```
Comment (AnikaAgiwal2711) [inline content/docs/storage/platforms/powerstore/Product_Guide.md:27]: same here

Comment (AnikaAgiwal2711) [reply content/docs/storage/platforms/powerstore/Product_Guide.md:27]: you can add space in between

File: content/docs/storage/platforms/powerstore/Product_Guide.md [hunk 4]
```diff
@@ -0,0 +1,1593 @@
...
+
+# Terraform Modules for Dell PowerStore
+--------------
+# Contents
+*   [Volume Resource](#volume-resource)
+    *   [Synopsis](#synopsis)
+    *   [List Of Parameters](#list-of-parameters)
+    *   [Examples](#examples)
+    *   [Notes](#notes)
+*   [SnapshotRule Resource](#snapshotrule-resource)
```
Comment (AnikaAgiwal2711) [inline content/docs/storage/platforms/powerstore/Product_Guide.md:17]: Snapshot Rule

File: content/docs/storage/platforms/powerstore/Product_Guide.md [hunk 5]
```diff
@@ -0,0 +1,1593 @@
...
+            <br>
+        </td>
+    </tr>
+    <tr>
+        <td>capacity_unit</td>
+        <td>The Capacity Unit corresponding to the size.</td>
+        <td>Optional and computed</td>
+        <td>String</td>
+        <td>GB</td>
+        <td>MB/GB/TB (256 TB max)</td>
```
Comment (AnikaAgiwal2711) [inline content/docs/storage/platforms/powerstore/Product_Guide.md]: It's better to add this as list

File: content/docs/storage/platforms/powerstore/Product_Guide.md (hunk 1 above)
Comment (AnikaAgiwal2711) [reply content/docs/storage/platforms/powerstore/Product_Guide.md]: This is aaplicable for all such instances

File: content/docs/storage/platforms/powerstore/Product_Guide.md (hunk 2 above)
Comment (prajwalpatil25) [reply content/docs/storage/platforms/powerstore/Product_Guide.md:22]: we have followed this naming convention everywhere. Confluence: https://confluence.cec.lab.emc.com/display/TFM/Resource+Schema+v1.0.0

File: content/docs/storage/platforms/powerstore/Product_Guide.md (hunk 1 above)
Comment (prajwalpatil25) [reply content/docs/storage/platforms/powerstore/Product_Guide.md]: done

File: content/docs/storage/platforms/powerstore/Product_Guide.md (hunk 4 above)
Comment (prajwalpatil25) [reply content/docs/storage/platforms/powerstore/Product_Guide.md:17]: we have followed this naming convention everywhere. Confluence: https://confluence.cec.lab.emc.com/display/TFM/Resource+Schema+v1.0.0

File: content/docs/storage/platforms/powerstore/Product_Guide.md (hunk 5 above)
Comment (prajwalpatil25) [reply content/docs/storage/platforms/powerstore/Product_Guide.md]: done

File: content/docs/storage/platforms/powerstore/Product_Guide.md (hunk 2 above)
Comment (AnikaAgiwal2711) [reply content/docs/storage/platforms/powerstore/Product_Guide.md:22]: Ok . No Problem

**Review by AnikaAgiwal2711 — APPROVED** [review]

**Pull request merged**